### ISSUE TITLE
As a data professional, I want to see a preview of a dataset visualisation before I download it so that I could see if it's useful

### ISSUE THREAD

**Issue by AnaBelgun** [TerriaJS/magda#302]
This task is just for putting the existing visualization code into the new dataset page. Other tasks are for making interactions (changing axes, scale etc) match the designs as they come through.

https://invis.io/HEFV4YNBG63

**Comment by tkeuneman** [comment]
- [ ] The ability to change the x/y axis of the visualisation
- [ ] The ability to change the visualisation
- [ ] The ability to give feedback if the visualisation is incorrect
- [ ] The ability to change scales

**Comment by tkeuneman** [comment]
- [ ] The ability to change the x/y axis of the visualisation
- [ ] The ability to change the visualisation
- [ ] The ability to give feedback if the visualisation is incorrect
- [ ] The ability to change scales

**Comment by AlexGilleran** [comment]
Separated out as #419, #420, #421

**Comment by tkeuneman** [comment]
Always picks the first visualisable distribution

**Comment by AlexGilleran** [comment]
Assigned to @tkeuneman for QA

**Comment by tkeuneman** [comment]
![screen shot 2018-03-13 at 11 18 30 am](https://user-images.githubusercontent.com/2201814/37315913-4d379bce-26b0-11e8-95e5-093669303882.png)
 - [ ] Show ten rows on the table before truncating.

![screen shot 2018-03-13 at 11 20 24 am](https://user-images.githubusercontent.com/2201814/37315967-8e18f7aa-26b0-11e8-8e21-30a059c4974e.png)
- [ ] Don't show the chart preview if there is no chart available. This does mean hiding the 'table' tab if there's only a table available.

@steve9164 I kinda feel like we should be able to create a chart for something like this yeah? http://search.data.gov.au/dataset/ds-dga-d39a9dc5-b3fe-4bd8-9ca1-c880bea27f86/details

Wasn't able to find one visualisable dataset :(

**Comment by steve9164** [comment]
That one's hard to parse. It has totals and normal values mixed, the date format is just six digits that you somehow have to make into a year and month.

**Comment by AlexGilleran** [comment]
Ok this one's got confusing - closing in favour of #672
